xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEBuffer.h

```diff
@@ -141,7 +141,7 @@ class CActiveAEBufferPoolAtempo : public CActiveAEBufferPool
   bool m_changeFilter;
   float m_tempo;
   int64_t m_lastSamplePts;
-  bool m_fillPackets;
+  bool m_fillPackets = false;
```

Comment:
Out of curiosity, should we not just init like the other bools in this struct? ie in the constructor. purely for consistency i would imagine?

https://github.com/xbmc/xbmc/blob/c9948be77a7e27416dfd025569182d747e9fd110/xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEBuffer.cpp#L464-L470

Reply:
You are right, this is a better option to fix this.

Reply:
I would prefer initialization in the class declaration. That way future initializations don't get missed.

It would be nice to update all the class members in a similar manner.

More work but sometimes "following the pattern" isn't always the best solution even if it is the easiest.